Tests/Application/Kernel.php

```diff
@@ -37,4 +37,15 @@ public function registerContainerConfiguration(LoaderInterface $loader): void
 
         $loader->load(__DIR__ . '/config/config_' . $this->getContext() . '.yml');
     }
+
+    protected function getKernelParameters(): array
+    {
+        $parameters = parent::getKernelParameters();
+
+        $gedmoReflection = new \ReflectionClass(\Gedmo\Exception::class);
+        $parameters['gedmo_directory'] = \dirname($gedmoReflection->getFileName());
+        $parameters['kernel.root_dir'] = __DIR__; // TODO remove when lowest version increased
```

Comment:
didn't find o ut who access this so I just did define it to make the CI lowest running work